Exiled.API/Features/Player.cs

```diff
@@ -1276,6 +1276,8 @@ public void Handcuff(Player cuffer)
             if (!IsCuffed && Vector3.Distance(Position, cuffer.Position) <= 130f)
             {
                 Cuffer = cuffer;
+                Inventory.SetDisarmedStatus(cuffer.Inventory);
+                new DisarmedPlayersListMessage(DisarmedPlayers.Entries).SendToAuthenticated();
```

Comment:
This should also be moved into `Cuffer`'s setter, so the entire thing will look like this.
```cs
if (value != null)
{
    Inventory.SetDisarmedStatus(value.Inventory);
    new DisarmedPlayersListMessage(DisarmedPlayers.Entries).SendToAuthenticated();
}

```